products.php

```diff
@@ -1,34 +1,35 @@
 <?php
-require_once ('vendor/autoload.php');
+require_once('vendor/autoload.php');
+
 use FurnitureStoreApi\Services\HeaderService;
 use FurnitureStoreApi\Products\ProductHydrator;
 use FurnitureStoreApi\Services\ResponseService;
 use FurnitureStoreApi\Categories\CategoryHydrator;
 use FurnitureStoreApi\Exceptions\InvalidCategoryException;
+use FurnitureStoreApi\Exceptions\InvalidCurrencyException;
+use FurnitureStoreApi\Services\CurrencyConversionClass;
+
 HeaderService::setHeaders();
 
-try
-{
-    if ($_GET['cat']>CategoryHydrator::getMaxCategory()||$_GET['cat']<0||!is_numeric($_GET['cat']))
-    {
+try {
+    if ($_GET['cat'] > CategoryHydrator::getMaxCategory() || $_GET['cat'] < 0 || !is_numeric($_GET['cat'])) {
```

Comment:
A bit counterintuitive that this is actually the opposite (NOT) case to the way you were checking the id before, you can see how this would be confusing.

Reply:
From my understanding of this, this is checking that if 'cat' is greater than max, or less than 0, or is not numeric, it throws an error.  So I have changed the other one to match.  But maybe that isn't the solution?